ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml

```diff
@@ -4542,17 +4542,6 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-4.12-upgrade-from-stable-4.12-aws-ipi-ovn-hypershift-inplace-f7
-  reporter_config:
```

Comment:
Those removal are unexpected

Reply:
checking, it is automatically done by `make update` on my local machine when solving conflicts.

Reply:
Those `reporter_config` are added explicitly (for slack notification).
`make update` will not remove it if it's already there.
If it's removed accidentally, e.g, while resolving conflicts, `make update` can't add it back, too.

Reply:
got it, not sure why they are removed during resolving conflicts.
Anyway I added them back, please help to review.